src/shared/networkrequest.cpp

```diff
@@ -200,8 +200,8 @@ void NetworkRequest::replyFinished() {
   logger.debug() << "Network reply received - status:" << status
                  << "- expected:" << expect;
 
-  QByteArray data = m_reply->readAll();
-  processData(m_reply->error(), m_reply->errorString(), status, data);
+  m_replyData.append(m_reply->readAll());
+  processData(m_reply->error(), m_reply->errorString(), status, m_replyData);
```

Comment:
You don't need to pass `m_replyData` to `processData`.

Reply:
Unfortunately, it seems that we do need to provide it as an argument for WASM